cluster/cluster.go

```diff
@@ -1,15 +1,84 @@
 package cluster
 
-import "github.com/lirm/aeron-go/aeron/idlestrategy"
+import (
+	"github.com/lirm/aeron-go/aeron/atomic"
+	"github.com/lirm/aeron-go/aeron/idlestrategy"
+	"github.com/lirm/aeron-go/cluster/codecs"
+)
 
 type Cluster interface {
+
+	// Position returns the position the log has reached in bytes as of the current message.
```

Comment:
If you follow the style guide, the comment should start with the function name always

```
LogPosition returns the position the log has reached in bytes as of the current message.
```

Reply:
Thanks for the feedback once again! Addressed!